src/game/Pet.h

```diff
@@ -143,8 +143,8 @@ extern const uint32 LevelStartLoyalty[6];
 
 #define ACTIVE_SPELLS_MAX           4
 
-#define PET_FOLLOW_DIST  1.0f
-#define PET_FOLLOW_ANGLE (M_PI_F/2.0f)
+#define PET_FOLLOW_DIST  2.00f
```

Comment:
My own initial testing tells me this change is delicious, but I kind of have to say that, I did the changes xD
No, yeah. you are completely right. This is almost 100% a visual thing, so it very much has to be felt and seen, rather than read.